pkg/peaks/docs/peaks_deployment/peaks-power-model-config.json

```diff
@@ -0,0 +1 @@
+{"node_1":{"k0":"<replace with node 1 k0>","k1":"<replace with node 1 k1>","k2":"<replace with node 1 k2>"},"node_2":{"k0":"<replace with node 2 k0>","k1":"<replace with node 2 k1>","k2":"<replace with node 2 k2>"}}
```

Comment:
could you define the schema definition of the model?

Reply:
We would like it to keep it flexible so that the power models are not limited to specific mathematical expressions (like `NodePower = K0 + K1 * e^(K2 * x)`). Ideally mathematical expression may also contain additional constant `K3` which is not possible with a fixed schema.